cmd/habitat/community/executor.go

```diff
@@ -147,5 +147,11 @@ func (e *CommunityExecutor) AddNode(update *state.StateUpdate) error {
 		return err
 	}
 
+	// Add node to list of data proxy peer nodes
+	err = e.node.DataProxy.AddPeerNode(transition.Node.Address)
```

Comment:
Nice. I think eventually we will create a better API for managing peer nodes, because several of our subsystems have their own node mappings that all are updated here.